### ISSUE TITLE
Connect API-Key to Category

### ISSUE THREAD

**Issue by streightedge** [jhuckaby/Cronicle#583]
I have not found anything in the documentation about limiting an API-Key just to a Category (only to a rule). Can this be done in Cronicle at the moment?

**Comment by jhuckaby** [comment]
Nope, only users can be limited to categories or server groups.  API keys cannot.  I'll look into adding this functionality in Cronicle v2 (will be released sometime in 2023).

Leaving this issue open.
